Sources/tart/Commands/Run.swift

```diff
@@ -69,12 +69,12 @@ struct Run: AsyncParsableCommand {
           Foundation.exit(1)
         }
       }
-
-      if noGraphics || vnc {
-        dispatchMain()
-      } else {
-        runUI()
-      }
+    }
+    
+    if noGraphics || vnc {
+      dispatchMain()
+    } else {
+      runUI()
     }
```

Comment:
Could you please check if #123 works for you? With this change it seems UI won't be shown at all because it will wait for the group to finish aka VM stops.

Reply:
Ventura was too problematic to me, restored to Monterey now. Will find a M1 machine to test that. Withdrawn the commit and leave it for now.

Reply:
Just tried #123 , works except Windowed mode